src/naming.jl

```diff
@@ -407,7 +410,7 @@ function parse_savename(filename::AbstractString;
     last_dot = findlast(".",savename_part)
     if last_dot == nothing || last_eq > last_dot
         # if no dot is after the last "="
-        # there is no suffix
+        # there is no suffixf
```

Comment:
Where did this change come from? suffix -> suffixf? I guess it's a typo that should be reversed?